addon/components/one-way-checkbox.js

```diff
@@ -15,29 +15,41 @@ const OneWayCheckboxComponent = Component.extend(DynamicAttributeBindings, {
   NON_ATTRIBUTE_BOUND_PROPS: ['update'],
 
   attributeBindings: [
-    'isChecked:checked',
     'type',
     'value'
   ],
 
+  checkedValue: computed('paramChecked', 'checked', function() {
+    let checkedValue = get(this, 'paramChecked');
+
+    if (checkedValue === undefined) {
+      return checkedValue = get(this, 'checked');
+    }
+
+    return checkedValue;
+  }),
+
   didInsertElement() {
     this._super(...arguments);
     this.element.addEventListener('click', (e) => this._click(e));
-  },
 
-  didReceiveAttrs() {
-    this._super(...arguments);
-
-    let value = get(this, 'paramChecked');
-    if (value === undefined) {
-      value = get(this, 'checked');
+    const checkedValue = get(this, 'checkedValue');
+    if (checkedValue) {
+      this.element.setAttribute('checked', 'checked');
```

Comment:
Set `checked="checked"` on the element if the initial value to this component is true